sources/packages/backend/apps/queue-consumers/src/services/cas-invoice-batch/cas-invoice-batch.service.ts

```diff
@@ -45,10 +55,14 @@ export class CASInvoiceBatchService {
     processSummary: ProcessSummary,
   ): Promise<CASInvoiceBatch | null> {
     return this.dataSource.transaction(async (entityManager) => {
+      const now = new Date();
+      // Gets the current fiscal year to be used (e.g. 2025) and adjust it to 2 digits only (e.g. 25).
+      const fiscalYear = getFiscalYear(now) - YEAR_ADJUSTMENT;
```

Comment:
Minor. Was there a reason to not just pick last 2 digits instead?. Not a blocker.